.tekton/rapidast-llm-push.yaml

```diff
@@ -32,14 +32,27 @@ spec:
     - PREFETCH=false
     - COMMIT_SHA={{revision}}
   taskRunSpecs:
+    - pipelineTaskName: clair-scan
+      stepSpecs:
+        - name: clair-scan-spec
+          computeResources:
+            requests:
+              cpu: '8'
+              memory: 16Gi
+            limits:
+              cpu: '16'
+              memory: 32Gi
     - pipelineTaskName: ecosystem-cert-preflight-checks
       stepSpecs:
         - name: app-check
           computeResources:
             requests:
-              memory: 512Mi
+              cpu: '8'
```

Comment:
That's a lot of CPU! I'm not sure how the big nodes are in konflux, but high request values may slow down the pipeline slightly if there is high competition for resources and it takes some for the require cpu value to be available on the node for scheduling.